.github/workflows/issue-release-workflow.yml

```diff
@@ -65,5 +65,5 @@ jobs:
           release-workflow \
           --branch-repo-token ${{ secrets.RELEASE_WORKFLOW_PUSH_SECRET }} \
           --issue-number ${{ github.event.issue.number }} \
-          --requested-by ${{ github.event.issue.user.login }} \
+          --requested-by ${{ (github.event.action == 'opened' && github.event.issue.body.user.login) || github.event.comment.user.login }} \
```

Comment:
Just skimming through the issue webhook event, is this `body` meant to be bere? It looks to be a string in https://docs.github.com/en/webhooks/webhook-events-and-payloads#issues

Reply:
Yes, it's meant to be there.  With the GitHub action syntax, the value of an expression is the last expression that is evaluated using short-circuit evaluation rules..  So what this does is that when we have an 'opened' event, the expression will evaluate to `github.event.issue.body.user.login` otherwise the expression will evaluate to `github.event.comment.user.login`.

Reply:
This is how you emulate a ternary conditional operator in Github Actions.

Reply:
I meant though that this changes `github.event.issue.user.login` to `github.event.issue.body.user.login`

Reply:
Ok, good catch.  I had that fix in another commit that I forgot to push.  Should be fixed now.